plugins/Monolog/config/config.php

```diff
@@ -97,6 +99,13 @@
     'Piwik\Plugins\Monolog\Handler\FileHandler' => DI\create()
         ->constructor(DI\get('log.file.filename'), DI\get('log.level.file'))
         ->method('setFormatter', DI\get('log.lineMessageFormatter.file')),
+    
+    '\Monolog\Handler\ErrorLogHandler' => DI\create()
+        ->constructorParameter('level', DI\get('log.level.errorlog'))
+        ->method('setFormatter', DI\get('log.lineMessageFormatter.file')),
+
+    '\Monolog\Handler\SyslogHandler' => DI\create()
+        ->constructor(DI\get('log.syslog.ident'), 'syslog', DI\get('log.level.syslog')),
```

Comment:
Apologies for the back and forth, but one last question here: was it intentional not to use `->method('setFormatter', DI\get('log.lineMessageFormatter.file'))` here?